docs/design/Non_Admin_Controller_design.md

```diff
@@ -0,0 +1,133 @@
+# Non-Admin Backup/Restore Design
+
+## Background
+OADP (Openshift API for Data Protection) Operator currently requires cluster admin access for performing Backup and Restore operations of applications deployed on the OpenShift platform. This design intends to enable the ability to perform Backup and Restore operations of their own application namespace for namespace owners aka non-admin users.
+
+## Goals
+- Enable non-admin backup operation
+- Enable non-admin restore operation
+
+## Non-Goals
+- Performance improvements of Backup and Restore Operations
+- Parallel Backup and Restore Operations
+
+## Use-Cases
+
+### Backup Operation
+- As a non-admin user/namespace owner with administrative priviledges for a particular namespace, the user should be able to:
+    - Create a Backup/Schedule of the namespace
```

Comment:
include the option to enable/disable non-admin schedules as an option please

Reply:
Do we need schedules at all for first iteration? To do schedules we'll need a NonAdminSchedule CR, etc. and the NAC will need to reconcile schedules, etc. Could we, for now, just direct users to create CronJobs which create NonAdminBackup CRs?

Reply:
> just direct users to create CronJobs

If it's going to greatly simplify scope and accelerate implementation then yes.

Reply:
@kaovilai well it eliminates the need to create a CRD and controller for NonAdminSchedules -- seems like a significant simplification if a CronJob meets our needs.

Reply:
we should tell upstream to get rid of schedules :P.. velero only checks at most once an hour anyway IIRC..